.circleci/config.yml

```
@@ -121,6 +121,9 @@ jobs:
             - run:
                 name: Run UI Tests
                 command: scl enable rh-nodejs6 "$XDMOD_SRC_DIR/tests/ui/runtests.sh --headless --log-junit ~/phpunit"
+            - run:
+                name: Ensure that no unexpected Apache errors were generated
+                command: test `fgrep -v ' [ssl:warn] ' /var/log/xdmod/apache-error.log | wc -l` = 0
```

Might be helpful to print the errors if there are any:
```suggestion
                command: >
                    errors=`fgrep -v ' [ssl:warn] ' /var/log/xdmod/apache-error.log`;
                    if [ -n "${errors}" ];
                    then
                        echo "${errors}";
                        false;
                    fi
```

---

This file is copied to the build artifacts (and hence the contents appear in the circle ci interface) so no need to redundantly output the errors twice.